src/at2/Human.java

```diff
@@ -0,0 +1,106 @@
+package at2;
+
+public abstract class Human {
+    private String name;
+    private int age;
+    private double height;
+    private double weight;
+    private String hairColor;
+
+    Human() {}
+
+    Human(String name, int age, double height, double weight, String hairColor) {
+        this.name = name;
+        this.age = age;
+        this.height = height;
+        this.weight = weight;
+        this.hairColor = hairColor;
+    }
+
+    abstract String getSex();
+
+    public void printInfo() {
+        String pronounce;
+        String pronounce2;
+        if (getSex() == "female") {
+            pronounce = "women";
+            pronounce2 = "She";
+        } else {
+            pronounce = "men";
+            pronounce2 = "He";
+        }
+        System.out.println("This " + pronounce + "'s name is " + name + ". " + pronounce2 + " is " + age);
+    }

```

Comment:
how can you rely on getSex() method in this class, while it is not implemented yet? What was the idea here? 
Could be discussed by voice